interfaces/aux_/aux_.yaml

```
@@ -20,6 +20,59 @@ components:
           description: Human-readable message indicating what error occurred and/or why.
           type: string
 
+    DSSInstancesResponse:
+      type: object
+      properties:
+        dss_instances:
+          type: array
+          items:
+            $ref: '#/components/schemas/DSSInstance'
+          default: []
+    DSSInstance:
+      type: object
+      properties:
+        instance_id:
```

Since we are already in `DSSInstance`:
```suggestion
        id:
```